app/helpers/i18n_helper.rb

```diff
@@ -1,6 +1,12 @@
 # frozen_string_literal: true
 
 module I18nHelper
+  def locales_available
+    OpenFoodNetwork::I18nConfig.available_locales.map do |locale|
```

Comment:
It's confusing to have `available_locales` and `locales_available`. What about `locale_options` in this case because it is the format for select input options?

Reply:
Applied in 07353de7870a5321ab26d30df57e1861b67ffe86